src/emulator/host/include/host/config.h

```diff
@@ -53,4 +53,5 @@ struct Config {
     std::string background_image = {};
     float background_alpha = 0.300f;
     int icon_size = 64;
+    optional<bool> discord_rich_presence = true;
```

Comment:
why optional?

Reply:
Why not? Would rather have more flexibility than less.

Reply:
What's the point of making it `optional` and directly initializing it with a value?

Reply:
The initialization is my bad, des asked for it to be optional and so i just did it. should i remove it?

Reply:
That is a very good point.

Then drop the ` = true` and check if the optional is empty [here](https://github.com/Vita3K/Vita3K/pull/519/files#diff-64550198e4afbca5fefcafa71ec8889fR341) and give it a default value if it is empty?

... or you can just make it mandatory. Up to you.